From 8986f0c9309c07dec31fc52df62dc739072d7cbb Mon Sep 17 00:00:00 2001
From: Iain Russell <Iain.Russell@ecmwf.int>
Date: Fri, 11 Oct 2019 12:00:03 +0100
Subject: [PATCH] CODESUI: fix path to Windows startup script from tarball

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index ab7bf3dc..18654b0c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -109,7 +109,7 @@ if(__external_build)
     set(SRC_PATH ${CMAKE_CURRENT_SOURCE_DIR}/src)
     set(SHARE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/share/codes_ui/etc)
     set(STARTUP_SCRIPT_TEMPLATE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/scripts/codes_ui_base.in")
-    set(WINDOWS_STARTUP_SCRIPT_PATH "${CMAKE_CURRENT_SOURCE_DIR}/scripts/codes_ui.bat")
+    set(WINDOWS_STARTUP_SCRIPT_PATH "${CMAKE_CURRENT_SOURCE_DIR}/codes_ui.bat")
     set(KEYPROFILE_PATH ${SHARE_PATH})
     set(VERSION_PATH ${SHARE_PATH})
     set(EXE_SRC_PATH ${CMAKE_CURRENT_SOURCE_DIR}/src/codes_ui)
-- 
2.13.7

